packages/components/inputs/time-input/src/time-input-body.tsx

```diff
@@ -1,4 +1,4 @@
-import PropTypes from 'prop-types';
+import { KeyboardEvent, MouseEvent } from 'react';
```

Comment:
```suggestion
import type { KeyboardEvent, MouseEvent } from 'react';
```

Reply:
👋🏼 Just for completeness depending on the TypeScript version one can now use `import { type foo } from './bar'` which means one doesn't have to split type imports.

I think we can stick with `import type` here and eventually on a Friday move everything to `import { type foo }` on a Friday. Did the same on `flopflip` [here](https://github.com/tdeekens/flopflip/pull/1558/files).

🔗 https://devblogs.microsoft.com/typescript/announcing-typescript-4-5/#type-on-import-names